### PR TITLE
change qt tarball file type to tat.xz

### PR DESCRIPTION
**Pull request by j8xixo12**
I found the file format in the zip file is **dos**, obviously zip is for Windows
tar.xz is for unix, therefore I change the file type from zip to tar.xz

### CODE REVIEW

File: scripts/build.d/qt [hunk 1]
```diff
@@ -11,14 +11,14 @@ if [ -z "${SYNCGIT}" ]; then
   pkgfull=$pkgname-$pkgver
   pkgfoler=qt-everywhere-src-$pkgver
 
-  pkgfn=$pkgfull.zip
-  pkgurl=https://download.qt.io/official_releases/qt/$pkgmajorver/$pkgver/single/qt-everywhere-src-$pkgver.zip
+  pkgfn=$pkgfull.tar.xz
+  pkgurl=https://download.qt.io/official_releases/qt/$pkgmajorver/$pkgver/single/qt-everywhere-src-$pkgver.tar.xz
 
   download_md5 $pkgfn $pkgurl $pkgmd5
```
Comment (yungyuc) [inline scripts/build.d/qt:17]: Why isn't the md5 hash updated?

Comment (j8xixo12) [reply scripts/build.d/qt:17]: ah… I forgot it.

Comment (j8xixo12) [reply scripts/build.d/qt:17]: And if md5 hash is wrong, `download_md5` seems not to terminate the build process right ?
it just show `md5 hash …. but got ….`

**Comment by j8xixo12** [comment]
I just updated the md5 hash.

**Comment by j8xixo12** [comment]
oh..wait, I push the wrong version.

File: scripts/build.d/qt (hunk 1 above)
Comment (yungyuc) [reply scripts/build.d/qt:17]: Yes, that's by design.  Many tarballs are large and hashing can only be calculated after it is fully downloaded.  It wastes too much bandwidth to discard the downloaded file or stop the process.

**Review by yungyuc — APPROVED** [review]

**Comment by yungyuc** [comment]
Feel free to merge

**Pull request merged**